network-store-client/src/main/java/com/powsybl/network/store/client/RestNetworkStoreClient.java

```diff
@@ -36,6 +36,7 @@ public class RestNetworkStoreClient implements NetworkStoreClient {
     private static final Logger LOGGER = LoggerFactory.getLogger(RestNetworkStoreClient.class);
 
     private static final int RESOURCES_CREATION_CHUNK_SIZE = 1000;
+    public static final String RETRYING = "Retrying...";
```

Comment:
maybe STR_RETRYING? to be consistent ?
```suggestion
    public static final String STR_RETRYING = "Retrying...";
```